codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -286,6 +286,12 @@ public class <%= schema_name %> {
                       this.<%= escape_reserved_word(field.camelize_name) %> = <%= escape_reserved_word(field.camelize_name) %>;
                       return this;
                   }
+
+                  // Unsets the <%= escape_reserved_word(field.camelize_name) %> property so that it is not serialized
+                  public void unset<%= field.classify_name %>() {
```

Comment:
It doesn't make sense to have this method for required fields, as is the case here.

Reply:
This makes sense. Will fix.

Reply:
Makes sense so I fixed this. 
This comment was on the optional field generator ☝️